control-plane/config/eventing-kafka-broker/200-controller/500-controller.yaml

```diff
@@ -69,24 +69,24 @@ spec:
             - name: SOURCE_DATA_PLANE_CONFIG_MAP_NAMESPACE
               value: knative-eventing
 
-            - name: BROKER_DATA_PLANE_CONFIG_MAP_NAME
+            - name: BROKER_CONTRACT_CONFIG_MAP_NAME
```

Comment:
I like that we rename them separately as a step before touching this section of code